website/config/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     proxyTable: {},
 
     // Various Dev Server settings
-    host: 'localhost', // can be overwritten by process.env.HOST
+    host: '0.0.0.0', // can be overwritten by process.env.HOST
```

Comment:
It’s not needed to change this. Per comment: `// can be overwritten by process.env.HOST`

To make it listen on `0.0.0.0`, you can set the environment variables:

```
set HOST=0.0.0.0
yarn dev
```

On Linux/Mac, you can set the environment variable when running the command:

```
HOST=0.0.0.0 yarn dev
```